ir/ir.def

```diff
@@ -243,6 +262,13 @@ class P4Table : Declaration, IAnnotated, IApply {
             ::error("%1% must be an expression", d);
             return nullptr; }
         return d->value->to<IR::ExpressionValue>()->expression; }
+    Constant getConstProp(cstring name) const {
```

Comment:
The word "constant" is overloaded, there are lots of constant values which are not integers.
So the naming is probably not the best.